module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -272,9 +274,10 @@ public function removeResourcesById(
     /**
      * Is this a public list?
      *
-     * @return bool
+     * @return     bool
+     * @deprecated This method is deprecated and will be removed once laminas is no longer used.
```

Comment:
Since we've decided to keep this method, you should remove the deprecation warning.